config/routes.rb

```diff
@@ -14,6 +14,8 @@
   namespace :admin do
     root to: 'pages#dashboard', as: :root
     resources :kudos
+    resources :categories
+    resources :category_rewards
```

Comment:
do we really need those routes?

Reply:
Yes, without it, no communication in views works.